src/settings/toml/tests/deploy_target.rs

```diff
@@ -0,0 +1,337 @@
+use super::wrangler_toml::WranglerToml;
+
+use crate::settings::toml::deploy_target::Zoned;
+use crate::settings::toml::route::Route;
+use crate::settings::toml::DeployTarget;
+use crate::settings::toml::Manifest;
+
+// TOP LEVEL TESTS
```

Comment:
if any of these descriptions are too cryptic, let me know and i can add to them. we can also add comments.